src/features/apply-form/components/form/RentalPeriodField.tsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from 'react';
+import { DateRange } from 'react-day-picker';
+import { useFormContext } from 'react-hook-form';
+
+import { getDateDistance, kstFormat } from '@toss/date';
+
+import {
+  Calendar,
+  FormControl,
+  FormField,
+  FormItem,
+  FormMessage,
+  Popover,
+  PopoverContent,
+  getAfterDays,
+} from '@/shared';
+
+import { DatePopoverTriggerButton } from '../button';
+
+type Props = {
+  name: string;
+};
+export const RentalPeriodField = ({ name }: Props) => {
+  const form = useFormContext();
+
+  const [date, setDate] = useState<DateRange | undefined>({
+    from: undefined,
+    to: undefined,
+  });
+
+  const currentDate = getAfterDays(7);
+
+  const onSelectCalendar = (from?: Date, to?: Date) => {
+    if (from) {
+      form.setValue('rentalStartDate', kstFormat(from, 'yyyy-LL-dd'));
+    }
+    if (from && to) {
+      form.setValue(
+        'rentalEndDate',
+        getDateDistance(from, to).days === 0 ? '' : kstFormat(to, 'yyyy-LL-dd'),
+      );
+    } else {
+      form.setValue('rentalEndDate', '');
+    }
+  };
+
+  const getMaxEndDate = (startDate?: Date) => {
+    return startDate
+      ? new Date(startDate.getTime() + 7 * 24 * 60 * 60 * 1000)
+      : undefined;
+  };
+
+  return (
+    <FormField
+      control={form.control}
+      name={name}
+      render={({ field }) => (
+        <FormItem className='flex w-full flex-col gap-2 border p-5 text-start'>
```

Comment:
```suggestion
        <FormItem className='flex w-full flex-col gap-2 border px-3 py-5 text-start'>
```
이렇게 돼야 하지 싶은데 한번 확인해부세용

Reply:
px-3으로 하게되면 다른 box들 보다 더 왼쪽에 위치하여 다시 p-5로 수정해두겠습니다~